types/office-js-preview/index.d.ts

```diff
@@ -16902,9 +16902,18 @@ declare namespace Office {
          */
         owner: String;
         /**
-         * The remote REST URL related to the owner’s mailbox.
+         * The REST API's base URL (currently https://outlook.office.com/api).
+         * Use with targetMailbox to construct REST operation's URL.
+         * 
+         * Example usage: `targetRestUrl + "/{api_version}/users/" + targetMailbox + "/{REST_operation}"`
+         */
+        targetRestUrl: String;
+        /**
+         * The target/owner's mailbox. Use with targetRestUrl to construct REST operation's URL.
+         * 
+         * Example usage: `targetRestUrl + "/{api_version}/users/" + targetMailbox + "/{REST_operation}"`
          */
-        restUrl: String;
+        targetMailbox: String;
```

Comment:
You definitely want `string` and not `String`, but I see the rest of the interface matches. It might be worth a follow-up PR to refactor all `String` -> `string`.

Reply:
I'll update that now.